shared/gh/partials/editable-parts.html

```diff
@@ -15,12 +15,12 @@
     <div class="row gh-row-part">
         <% _.each(data, function(part) { %>
             <div class="col-xs-4 gh-part">
-                <% if (part.isEditing) { %>
+                <% if (part.isEditing && (part.isEditing.displayName !== gh.data.me.displayName)) { %>
```

Comment:
Do we get an ID on the isEditing object? If so we should use that as display names can be duplicated.